### ISSUE TITLE
Quasar dev -> Compiling of "UI" - error

### ISSUE THREAD

**Issue by softal** [quasarframework/quasar#9092]
After fresh installation of Qasar 2.0 application I encountered an error:
```
 App • Chaining "UI" Webpack config
 App •  WAIT  • Compiling of "UI" in progress...
(node:8208) UnhandledPromiseRejectionWarning:   TypeError: The 'compilation' argument must be an instance of Compilation
...
Use `node --trace-warnings ...` to show where the warning was created)
(node:8208) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8208) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```
**To Reproduce**
1. create a project folder with Quasar CLI:
   $ quasar create <folder_name> --branch next
2. go to <folder_name>
3. run quasar dev

**Screenshots**

![image](https://user-images.githubusercontent.com/5954367/116522486-88b51000-a8d5-11eb-9073-50c56a6aadc9.png)
![image](https://user-images.githubusercontent.com/5954367/116522751-d3cf2300-a8d5-11eb-82c9-02291d8eb425.png)

**Platform (please complete the following information):**
Quasar Version:
@quasar/app Version:
Dev mode.......... spa
 Pkg quasar........ v2.0.0-beta.14
 Pkg @quasar/app... v3.0.0-beta.18
 Pkg webpack....... v5
 Debugging......... enabled

Operating System - Windows_NT(10.0.17763) - win32/x64
NodeJs - 14.15.1

Global packages
  NPM - 7.5.2
  @quasar/cli - 1.1.3

**Comment by hawkeye64** [comment]
@softal Can you try with yarn and see if you get different results?

**Comment by francoism90** [comment]
@hawkeye64 I'm having the error on NPM, how can you move to yarn - simple use `yarn install`?

**Comment by hawkeye64** [comment]
@francoism90 That's good to hear! We always recommend using yarn over npm for local packages. It just seems to "work better".

**Comment by therealcoder1337** [comment]
can confirm this is an issue with npm 7, but not with npm 6. yarn also works.
i've tried deleting `.quasar`, `node_modules` and even my global modules, but it still didn't work with npm 7.

**Comment by jonathanpmartins** [comment]
> can confirm this is an issue with npm 7, but not with npm 6. yarn also works.
> i've tried deleting `.quasar`, `node_modules` and even my global modules, but it still didn't work with npm 7.

I don't thing this is a NPM 7 issue. This problem was possible introduced with the ultimate release of the beta version (remember is in development).

I have an app that worked fine until I tried to update to last beta. It broke with this same error! I think this problem was introduced in `beta.16`. Try to update you `package.json` file and force the `3.0.0-beta.15` version. Perhaps it will solve your problem!

Working with NPM 7:
```json
"devDependencies": {
  "@quasar/app": "3.0.0-beta.15",
},
```

Not Working with NPM 7:
```json
"devDependencies": {
  "@quasar/app": "3.0.0-beta.20",
},
```

**Comment by IlCallo** [comment]
Can you try using npm 7.11.2? I just tried it and it works oob

**Comment by jonathanpmartins** [comment]
Yes, the error happens with 7.11.2. I explain the step by step on how I got there here: https://github.com/quasarframework/quasar/issues/9180#issuecomment-834052556

**Comment by hawkeye64** [comment]
@jonathanpmartins Try this: https://github.com/quasarframework/quasar/issues/9180#issuecomment-833673423

**Comment by rstoenescu** [comment]
Pushed a workaround for an issue in npm v7.
Available in "@quasar/app" v3.0.0-beta.21 (just released it now).
